### ISSUE TITLE
tether.io

### ISSUE THREAD

**Issue by rwwagner90** [shipshapecode/tether#312]
@TrevorBurnham do you think it would be possible to get tether.io transferred to us as well? As we iterate, we will want to update docs.

**Comment by TrevorBurnham** [comment]
@rwwagner90 Ah, I believe all you need to do is update the [gh-pages](https://github.com/shipshapecode/tether/tree/gh-pages) branch and the site will rebuild. I can look into transferring the domain name itself over to you as well.

**Comment by rwwagner90** [comment]
@TrevorBurnham yeah, I was able to run the hsdoc command, so we are okay for awhile. I'm just thinking as we go forward, and docs evolve, it could be good to have the domain for flexibility.

**Comment by rwwagner90** [comment]
@TrevorBurnham I am having some issues building the docs now and things are broken. Would you be able to help?

**Comment by TrevorBurnham** [comment]
What kinds of issues building the docs? I'm happy to help if I can, but no one (including me) has looked at the doc-building script in years. 😬

**Comment by rwwagner90** [comment]
@TrevorBurnham I'm able to build and publish but there are a couple issues. One is tether.io needs some DNS or verification or something to be used by gh-pages, and I believe github.hubspot.com is no longer a thing as well.

**Comment by TrevorBurnham** [comment]
Hmm. If you replace e.g.

```
https://github.hubspot.com/tether/resources/executr/lib/CodeMirror/codemirror.js
```

with

```
http://tether.io/resources/executr/lib/CodeMirror/codemirror.js
```

then it should work. Unfortunately, `https` to tether.io won't work because it doesn't have a valid cert. You'll be able to generate one once you have control over the domain name. The cogs within HubSpot are still churning on that front…

**Comment by erkie** [comment]
Use this gist to try it out locally:

https://gist.github.com/erkie/946e5f5932767420970e80f3583d759d

**Comment by rwwagner90** [comment]
@erkie I updated the links, but not all of them worked.

**Comment by bobc7i** [comment]
>do you think it would be possible to get tether.io transferred to us as well?

@rwwagner90 Sorry for the long delay on this. Are you still interested in doing the tether.io transfer? HubSpot is ready to move forward on this now.

**Comment by rwwagner90** [comment]
@bobc7i that would be great! We've been unable to debug properly and I think having it would help.

**Comment by bobc7i** [comment]
Great @rwwagner90 , I will email you the transfer agreement doc to sign.
